eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -32,6 +32,11 @@ jobs:
         displayName: 'Use Python 3.6'
         inputs:
           versionSpec: '3.6'
+      - task: UseDotNet@2
```

Comment:
What does this doe for us?

Reply:
Installs 5.0, it's not on agents right now.

Reply:
Ok this will likely start to cause some reliability issues that we were seeing in the past, so we should keep an eye on this. Looks like they are working on .NET 5 support https://github.com/actions/virtual-environments/issues/1891.